kafka-client/src/main/resources/license-keys/license-keys.json

```diff
@@ -0,0 +1 @@
+{"keys":  []}
```

Comment:
I guess when we create the keys well update it here?

Reply:
yep, that's right!